src/guide/frontend/android.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Android Client SDK
+---
+
+# Android Client SDK
+
+The Passwordless.dev Android client SDK gives users the ability to leverage their device’s built-in fingerprint sensor and/or FIDO security keys for secure passwordless access to websites and native applications that support the FIDO2 protocols
+
+## Creating your first Android application
+
+### Requirements
+
+- Android 7.0 (API level 24) or higher
```

Comment:
I don't think this version of Android supports Passkeys, I think device bound passkeys were added on Android 9 and the CredMan "modern passkeys were added in Android 14.

https://developers.google.com/identity/passkeys/supported-environments